rest/config.go

```diff
@@ -414,6 +438,25 @@ func (dbConfig DbConfig) validate() error {
 		}
 	}
 
+	if dbConfig.CacheConfig != nil {
+
+		if dbConfig.CacheConfig.ChannelCacheConfig != nil {
+			if dbConfig.CacheConfig.ChannelCacheConfig.MaxNumber != nil {
+				if *dbConfig.CacheConfig.ChannelCacheConfig.MaxNumber < 1 {
+					return fmt.Errorf("minimum value for max_number is 1")
+				}
+			}
+		}
+
+		if dbConfig.CacheConfig.RevCacheConfig != nil {
+			if dbConfig.CacheConfig.RevCacheConfig.ShardCount != nil {
+				if *dbConfig.CacheConfig.RevCacheConfig.ShardCount < 1 {
+					return fmt.Errorf("minimum value for shard_count is 1")
```

Comment:
Make the error message rev cache config specific, so the user knows exactly where this option is.